src/world_interface/real_world_constants.h

```diff
@@ -98,7 +98,6 @@ constexpr auto motorSerialIDMap = frozen::make_unordered_map<motorid_t, can::dev
 	 {motorid_t::shoulder, DEVICE_SERIAL_MOTOR_SHOULDER},
 	 {motorid_t::elbow, DEVICE_SERIAL_MOTOR_ELBOW},
 	 {motorid_t::forearm, DEVICE_SERIAL_MOTOR_FOREARM},
-	 {motorid_t::wrist, DEVICE_SERIAL_MOTOR_WRIST},
```

Comment:
We need to add entries here for `wristDiffLeft` and `wristDiffRight`. Note sure about the exact serial ids though I'll have to get back to you. For now, put a dummy value (set both to `DEVICE_SERIAL_MOTOR_WRIST` and leave a TODO note)